### DISCUSSION TITLE
Deleting photos in iOS app doesn't remove them from local/native photo roll anymore

### DISCUSSION THREAD

**Discussion by Qhilm** [immich-app/immich#14443] [Q&A]
It stopped working a month ago I would say, roughly.

I did check that Immich has "full access" to photos under settings > privacy. I don't see how it could be any different anyway, since syn is working, all photos are visible in the Immich app. Only the "delete" behaviour is not working as expected...

**Comment by Qhilm** [comment]
I uninstalled and reinstalled the Immich app, but it's still the same. It's unable to delete the photos from the local roll now...

**Reply by alextran1502** [reply]
Can you do a screen capture on how you perform the delete operation?

**Reply by Qhilm** [reply]
thanks for getting back to me.

When opening a picture in the iOS app, below it, there's "share", "edit", "archive", and  "delete".

I tap the "delete" button and it doesn't ask for "can I delete the native picture in your local camera roll" (I don't remember the exact wording of that prompt, but that's a native iOS prompt).

But it appears sometimes it works, sometimes it does not, at least since I reinstalled the app. I just tested two pictures, both of which are from my camera roll and one got deleted (I saw the prompt), the other one did not (no prompt for deleting from camera roll).

I will continue testing and try to figure out what the pattern is.

**Reply by alextran1502** [reply]
That delete button will only put the photo in trash, you will need to go to the Trash screen from the Library page, clear the trash would prompt a permanent delete question from native iOS side

**Reply by Qhilm** [reply]
Thanks for this, that's helpful, I cannot reproduce it anymore but I am 100% certain that deleting picture from the main timeline was causing the popup and pictures were removed from the phone's camera roll as well.

I had it again yesterday. Assuming I can reproduce this, which logs should I enable to best troubleshoot this?
